.appveyor.yml

```diff
@@ -0,0 +1,46 @@
+# See https://www.appveyor.com/docs/getting-started-with-appveyor-for-linux/
+skip_branch_with_pr: true
+
+# AppVeyor will run on the output branch commits until the commit message
+# contains [ci skip]
+# Enable 'Do not build on "Push" events' in the AppVeyor project settings
+# to only build commits from pull requests
+branches:
+  only:
+    - master
+
+only_commits:
+  files:
+    - content/
```

Comment:
This is cool but perhaps too restrictive. For example, don't we want to test that changes to environment.yml or .appveyor.yml don't break AppVeyor builds?

Reply:
Yes, I wanted to demonstrate the possibility but am undecided on the best behavior.  I want to avoid notification fatigue.  I'm also assuming we'll rely on Travis CI builds for most of our traditional continuous integration and use AppVeyor only for pull request review.

An intermediate option would be to add more build-related files to this whitelist.  We should be able to run AppVeyor if the content, `environment.yml`, `.appveyor.yml`, or any `.sh` files in the repository change.

Reply:
I didn't think it is necessary to run AppVeyor if `ci/deploy.sh` changes because that is only used by Travis CI.  I also want to use the `.appveyor.yml` to explain or document these whitelisted files so that we don't have to maintain a separate list in `SETUP.md`.